### PR TITLE
Add checkpoints panel to debug view

### PR DESCRIPTION
**Pull request by DatGuyJonathan**
### What does this PR do?
image

It's only an empty view. The ability to add stuff to it comes later. 

### What issues does this PR fix or reference?
@W-4719033@

### CODE REVIEW

File: packages/salesforcedx-vscode-replay-debugger/package.nls.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "view_checkpoints": "Checkpoints",
```
Comment (DatGuyJonathan) [inline packages/salesforcedx-vscode-replay-debugger/package.nls.json]: @ruthemmanuelle , this is the panel name. We're calling it checkpoints to match what Developer Console calls it: https://help.salesforce.com/articleView?id=code_dev_console_checkpoints_setting.htm&type=5

File: packages/salesforcedx-vscode-replay-debugger/src/breakpoints/checkpointService.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright (c) 2018, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import { Event, EventEmitter, TreeDataProvider, TreeItem } from 'vscode';
+
+export class CheckpointService implements TreeDataProvider<BaseNode> {
```
Comment (DatGuyJonathan) [inline packages/salesforcedx-vscode-replay-debugger/src/breakpoints/checkpointService.ts]: Just a skeleton - nothing can configure tree items in the checkpoints panel yet.

File: packages/salesforcedx-vscode-replay-debugger/package.json [hunk 3]
```diff
@@ -72,6 +73,15 @@
         "language": "apex"
       }
     ],
+    "views": {
+      "debug": [
+        {
+          "id": "sfdx.force.view.checkpoint",
+          "name": "%view_checkpoints%",
+          "when": "sfdx:project_opened"
```
Comment (DatGuyJonathan) [inline packages/salesforcedx-vscode-replay-debugger/package.json]: Checkpoints panel is only visible when salesforcedx-vscode-core extension sets this context.

File: packages/salesforcedx-vscode-replay-debugger/package.nls.json (hunk 1 above)
Comment (ruthemmanuelle) [reply packages/salesforcedx-vscode-replay-debugger/package.nls.json]: Sounds great.

**Pull request merged**